cmd/merge-action/main.go

```diff
@@ -82,6 +82,14 @@ func main() {
 
 	if !inTargetRelease(branchToTargetRelease[prBaseBranch], bug.TargetRelease) {
 		fmt.Printf("Merging fix for the branch: %v which is not associated with the target release: %v", prBaseBranch, bug.TargetRelease)
+		body := fmt.Sprintf("Unable to find the base branch: %s in target releases for bug: %s", branchToTargetRelease[prBaseBranch], strings.Join(bug.TargetRelease, ", "))
```

Comment:
I think this might be misleading shouldn't it say that the current merge is not for the target release?